Assets/GothicVR/Scripts/Creator/MusicCreator.cs

```diff
@@ -42,6 +42,13 @@ public enum Tags : byte
 
         private static GameObject backgroundMusic;
 
+        private float updateInterval = 5f; // Update music every 5 seconds
+        private float timer = 0f;
+
+        // This multiplier is used to increase the buffer size and reduce the number timess PrepareData is called
+        // also affects the delay of the music, it doesnt sound so harsh when switching
+        private int bufferSizeMultiplier = 16;
+
         void Start()
         {
             if (!FeatureFlags.I.EnableMusic)
```

Comment:
Maybe related to another topic, but: When I start the game, I get the following warning:
![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/cf46c100-d2db-4073-acab-1ca292939ef1)

Can you fix it with this branch as well? Should be a quick win.

Reply:
And when I start main menu, I get this:
![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/7532557d-422b-449d-90f0-e20cccbda9e3)

Reply:
we can ignore the no audio listeners log, but the multiple listeners could be because you left another scene except bootstrap loaded

Reply:
Please don't I would love to have our logs clean as long as possible.
Should be an easy thing to fix, right?